ApiTest.Business/Models/User.cs

```diff
@@ -0,0 +1,29 @@
+namespace ApiTest.Business.Models
+{
+    public class User
+    {
+        public int Id { get; set; }
+        public string Name { get; set; } = string.Empty;
+        public string Username { get; set; } = string.Empty;
+        public string Email { get; set; } = string.Empty;
+        public Address Address { get; set; } = new Address();
+        public string Phone { get; set; } = string.Empty;
+        public string Website { get; set; } = string.Empty;
+        public Company Company { get; set; } = new Company();
+    }
+
+    public class Address
+    {
+        public string Street { get; set; } = string.Empty;
```

Comment:
why do u need to init the props with the values?
they will be initialized by default whil creating instance of module
if the value is not set it will be null by default (if string)
int = 0 by default, u can use ? operator (int? or string?) to make the props nullable if required
it should not break anything when u dont need to send a street - that null is sent or the json field is absent